### ISSUE TITLE
WordPress Coding Standards

### ISSUE THREAD

**Issue by picandocodigo** [picandocodigo/List-Category-Posts#62]
Fix the code for the WordPress Coding Standards:
http://codex.wordpress.org/WordPress_Coding_Standards

**Comment by zymeth25** [comment]
I have this set up in the GUI repo, with automatic [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) checks in Travis and Vagrant, using [WPCS rules](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards).

Most of the code can be automatically fixed with the `phpcbf` tool. If you are still interested I can implement it here as well. WPCS has indentation with tabs but it can be customized to 2 spaces as we use here.

**Comment by zymeth25** [comment]
I have a branch with some stuff that deals with it.

**Comment by picandocodigo** [comment]
@zymeth25 I've been trying to get rid of the `if: endif;` and `foreach: endforeach;` blocks, don't know why I used to think that's a good idea, you can't make PHP look like Ruby :laughing:. And also changing things I thought were ok at the time but I don't like now. I don't think we need to adhere 100% to the WordPress standards, some things I don't agree with, but we should at least have an internal standard that we adhere to for concistency. What do you think?

**Comment by zymeth25** [comment]
Yes I think that an internal standard is a good idea. We can just edit WPCS and adjust the rules we don't like. And then it can be automated with PHP Code Sniffer.
